app/views/users/show.html.erb

```diff
@@ -7,10 +7,13 @@
 <h2>Viewing Parties</h2>
 
 <section id="hosting-viewing-parties">
-  <h3>Movie Title:</h3><br>
-  <h3>Date and Time of Event:</h3><br>
-  <h3>Hosting: <%= current_user.email %></h3><br>
-  <h3>Invited:</h3><br>
+  <% @user_hosting_parties.each do |party| %>
+    <h4>Movie Title: <%= party.movie_title %></h4><br>
+    <p>Date and Time of Event: <%= party.date %></p><br>
+    <p>Duration: <%= party.duration %></p><br>
+    <p>Hosting: <%= current_user.email %></p><br>
+    <p>Invited:</p><br>
+  <% end %>
```

Comment:
Fantastic! Once we got the movies and parties info here, we were able to get the info necessary for these sections built out. We can now connect and add the friends for the `Invited` section.